### PR TITLE
Add support for in_content, sidebar and sticky_sidebar

### PR DESCRIPTION
**Pull request by nicomvaldez**
This PR includes the support for the new units:
- in_content
- sidebar
- sticky_sidebar

The tweak was straightforward, and the best way for testing it is running the react app included in the demos folder, the commands for running it are in the readme file

Once this PR is approved we have to publish it here: https://www.npmjs.com/package/@playwire/pw-react-component

### CODE REVIEW

**Review by tcreamer1 — APPROVED** [review]
PR looks good. Noting that when corner_ad_video fully rolls out, we will need to do the same there.

File: demos/pw-react-app/yarn.lock [hunk 1]
```diff
@@ -1068,9 +1068,8 @@
   "resolved" "https://registry.npmjs.org/@nodelib/fs.stat/-/fs.stat-1.1.3.tgz"
   "version" "1.1.3"
 
-"@playwire/pw-react-component@^1.0.7":
-  "integrity" "sha512-lv8ftztSjlLt7lAEF4/z9kMoT2oCISLQxsgBFeTEmg4xGyaH2PvgryOyRZLoKhJtcZai/3jts7IaipPOACMgbA=="
-  "resolved" "https://registry.npmjs.org/@playwire/pw-react-component/-/pw-react-component-1.0.7.tgz"
+"@playwire/pw-react-component@file:../..":
```
Comment (vitkrv) [inline demos/pw-react-app/yarn.lock:1071]: Is it ok for our case - having an URL to local file instead of repository URL?

Comment (nicomvaldez) [reply demos/pw-react-app/yarn.lock:1071]: Correct, that is only for testing during code review, I will update it when publishing it to npm registry, @vitkrv @tcreamer1

**Review by mjquintas — APPROVED** [review]
Looks good!

**Pull request merged**